aleapp.py

```diff
@@ -12,134 +13,122 @@
 from tarfile import TarFile
 
 parser = argparse.ArgumentParser(description='ALEAPP: Android Logs, Events, and Protobuf Parser.')
-parser.add_argument('-o', choices=['fs','tar', 'zip'], required=True, action="store",help="Directory path, TAR, or ZIP filename and path(required).")
+parser.add_argument('-o', choices=['fs','tar','zip'], required=True, action="store",help="Input type (fs = extracted to file system folder)")
 parser.add_argument('pathtodir',help='Path to directory')
 
-# if len(sys.argv[1:])==0:
-# 	parser.logfunc_help()
-# 	parser.exit()
-
 start = process_time()
-	
+    
 args = parser.parse_args()
 
 pathto = args.pathtodir
 extracttype = args.o
 start = process_time()
 
-tosearch = {'wellbeing': '*/com.google.android.apps.wellbeing/databases/*',
-			'wellbeingaccount':'*/com.google.android.apps.wellbeing/files/AccountData.pb',
-			'usagestats':'*/usagestats/*',
-			'recentactivity':'*/system_ce/*'}
-'''
-tosearch = {'redditusers':'*Data/Application/*/Documents/*/accounts/*',
-			'redditchats':'*Data/Application/*/Documents/*/accountData/*/chat/*/chat.sqlite'}
-'''
-	
 os.makedirs(reportfolderbase)
-os.makedirs(reportfolderbase+'Script Logs')
+os.makedirs(os.path.join(reportfolderbase, 'Script Logs'))
```

Comment:
Awesome, this is the right way to handle file paths, I'll create an issue in iLEAPP as well to do the same there.